src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/fingerprints/ShortsPlayerConstructorFingerprint.kt

```diff
@@ -33,6 +36,18 @@ object ShortsPlayerConstructorFingerprint : MethodFingerprint(
         Opcode.IPUT_OBJECT,
         Opcode.NEW_INSTANCE,
         Opcode.INVOKE_DIRECT,
-        Opcode.IPUT_OBJECT
-    )
-)
\ No newline at end of file
+        Opcode.IPUT_OBJECT,
+        Opcode.IPUT_OBJECT,
+        Opcode.CONST_4
+    ),
+    customFingerprint = { methodDef ->
+        methodDef.implementation?.instructions?.any { instruction ->
+            (instruction as? WideLiteralInstruction)?.wideLiteral ==
+                    targetResourceId
+        } == true
+    }
+)
+
+val targetResourceId = ResourceIdMappingProviderResourcePatch.resourceMappings.single {
+    it.type == "id" && it.name == "reel_persistent_edu_button_group"
+}.id
```

Comment:
This is dangerous, since it can not be guaranteed, that the `ResourceIdMappingProviderResourcePatch` has been executed before this code runs.